repair/row_level.cc

```diff
@@ -1731,7 +1731,8 @@ class repair_meta {
             auto sz = set_diff.size();
             return get_row_diff(std::move(set_diff), needs_all_rows).then([this, remote_node, sz] (std::list<repair_row> row_diff) {
                 if (row_diff.size() != sz) {
-                    throw std::runtime_error("row_diff.size() != set_diff.size()");
+                    rlogger.warn("Hash conflict detected, keyspace={}, table={}, range={}, row_diff.size={}, set_diff.size={}",
+                            _schema->ks_name(), _schema->cf_name(), _range, row_diff.size(), sz);
```

Comment:
What is the user supposed to do if they see the warning? I suggest to drop it to debug, or maybe to info but with an explanation that no action is needed.

Reply:
The use can rerun the repair. Since each repair job will use a random hash seed, hopefully next time the hash conflict is gone.  

I am thinking about detecting conflicts and fallback to sync all the rows inside the row buffer which is 32MB in size, or fallback to a hasher with more bits.

Reply:
In this particular case, the user needs no action because node2 is perfectly repaired (got the missing row3 and row3').  In other cases,  like:

```
    Node1:
    row1  -> hash1
    row2  -> hash2
    row3  -> hash3
   
    Node2:
    row1  -> hash1
    row2  -> hash2  
    row3' -> hash3
```
n1 will not ask n2 to send row with hash3. So row3 and row3' will not be repaired in this run.

Reply:
Well, need to explain more in the message. Some of our users are not experts in row-level repair.

Reply:
OK. I added more in the message in the new version.